react/RecommendationBuyTogether.tsx

```diff
@@ -45,7 +47,9 @@ const RecommendationBuyTogether: StorefrontFunctionComponent<Props> = ({
   }, [error, data])
 
   return recommendations && recommendations.length > 0 ? (
```

Comment:
```suggestion
  return recommendations?.length > 0 ? (
```

Reply:
ts does not allow me to do this 'cause I am using it for a comparison (`> 0`) but I will do it without this comparison since the result is the same